.travis.yml

```diff
@@ -20,11 +20,11 @@ addons:
 
 before_install:
   - if [ ! -d $HOME/gcloud/google-cloud-sdk ]; then
-        mkdir -p $HOME/gcloud
-        wget https://dl.google.com/dl/cloudsdk/release/google-cloud-sdk.tar.gz --directory-prefix=$HOME/gcloud
-        cd $HOME/gcloud
-        tar xzf google-cloud-sdk.tar.gz
-        ./google-cloud-sdk/install.sh -q
+        mkdir -p $HOME/gcloud; 
```

Comment:
I _think_ that a less punctuationful way of doing this is with a _literal block_ entry:

```yaml
  - |
    if [ -d ....]; ...
       blah
       blah
    fi
 - next thing
```